.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "libraries/Adafruit-PWM-Servo-Driver-Library"]
-	path = libraries/Adafruit-PWM-Servo-Driver-Library
+	path = libraries/libraries/Adafruit-PWM-Servo-Driver-Library/Adafruit-PWM-Servo-Driver-Library
```

Comment:
The double `libraries` is necessary because that's the path the tool looks in, for some reason. I fear the duplicate `Adafruit...` directory was me messing up a `git mv`